src/views/partials/blogs/create.hbs

```diff
@@ -0,0 +1,20 @@
+<form action="/blogs/create" method="post" autocomplete="off" enctype="application/json">
+  <input name="id" type="hidden" value=" ">
+  <div class="form-group">
+    <label for="title">Title:</label>
+    <input type="text" class="form-control" id="title" name="title" style="width:700px" required>
+  </div>
+  <input id="today" name="date" type="hidden" value=" ">
+  <div class="form-group">
+    <label for="author">Author:</label>
+    <input type="text" class="form-control" id="title" name="author" style="width:700px" required>
+  </div>
+  <div class="form-group">
+    <label for="content">Content:</label>
+    <textarea name="content" class="form-control" id="content" style="width:700px; height:300px;">
+  </textarea>
+  </div>
+  <input name="imagelink" type="hidden" value=" ">
```

Comment:
Regarding this and other hidden inputs, why are they here? Maybe it is better to add date on the server side?
And regarding the imagelink, maybe it should not be hidden?

Reply:
I just stuck to your instructions to create a form with three inputs (title, author, content), as described in the trello card. However, in the blogpost.js file id, date, and imagelink are set to required, so I needed some kind of dummy.

So, shall we add the date on the server side and provide the option to attach an image link in the form?

Reply:
@pheeria Shall I add another field to attach an image link? I guess we are not far enough yet to actually add an upload function since our mLab storage is too small to include images. Do we have a long term solution for that?

Reply:
For now, please ignore the image link. We don't need it yet :)